src/client/Client.cc

```diff
@@ -5956,7 +5956,11 @@ void Client::unmount()
   ldout(cct, 2) << "unmounted." << dendl;
 }
 
-
+void Client::unmount()
+{
+  Mutex::Locker lock(client_lock);
```

Comment:
not remove "Mutex::Locker lock(client_lock)" in Client::_umount ?

Reply:
Oh my, good catch -- will fix.